src/containers/PageConstructor/components/ConstructorBlocks/ConstructorBlocks.tsx

```diff
@@ -57,16 +57,14 @@ export const ConstructorBlocks = ({items, shouldRenderBlock}: ConstructorBlocksP
             }
 
             itemElement = (
-                <BlockIdContext.Provider value={blockId}>
-                    <ConstructorItem data={item} key={blockId}>
-                        {children}
-                    </ConstructorItem>
+                <BlockIdContext.Provider value={blockId} key={blockId}>
```

Comment:
what is the meaning of context provider `key` attribute?

Reply:
it is usual key prop of a child component